### PR TITLE
Add samples for BigInt typed array API

### PR DESCRIPTION
**Pull request by Elchi3**
Examples for these pages:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView/getBigInt64
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView/getBigUint64
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView/setBigInt64
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView/setBigUint64

These example are very trivial and maybe not very useful, vut so are the other dataview getter/setter examples. Let me know if you have ideas and want me to try harder.

Looks better with https://github.com/mdn/bob/pull/254 which I spent more time on than the examples itself.

### CODE REVIEW

**Comment by wbamberg** [comment]
I don't mind these being boring but is there a reason not to use bigger integers, to help emphasize the motivation for using these types?

For example:

```js
// create an ArrayBuffer with a size in bytes
var buffer = new ArrayBuffer(16);

var view = new DataView(buffer);
view.setUint16(0, 65536);     // (max unsigned 16-bit integer + 1)
view.setBigUint64(8, 65536n);

console.log(view.getUint16(0));
// expected output: 0

console.log(view.getBigUint64(8));
// expected output: 65536n
```

**Comment by Elchi3** [comment]
Thanks, Will! That makes sense to me. In the documentation, I've now also written down the max values and what happens when there is overflow. Really valuable information to have, I think.

File: live-examples/js-examples/dataview/dataview-getbigint64.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">// create an ArrayBuffer with a size in bytes
+var buffer = new ArrayBuffer(16);
+// Highest possible BigInt value that fits in a signed 64-bit integer
+const max = 2n ** (64n -1n) - 1n;
```
Comment (wbamberg) [inline live-examples/js-examples/dataview/dataview-getbigint64.html]: It's odd to mix `var` and `const` like this. Can we use `const` throughout?

File: live-examples/js-examples/dataview/dataview-getbigint64.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">// create an ArrayBuffer with a size in bytes
+var buffer = new ArrayBuffer(16);
```
Comment (wbamberg) [inline live-examples/js-examples/dataview/dataview-getbigint64.html]: I know this is amazingly pedantic even for me but I would like a blank line between this one and the next.

File: live-examples/js-examples/dataview/dataview-getbigint64.html (hunk 1 above)
Comment (Elchi3) [reply live-examples/js-examples/dataview/dataview-getbigint64.html]: Agreed. This was me just copy-pasting from the existing samples and not thinking. Good catch! :)

File: live-examples/js-examples/dataview/dataview-getbigint64.html (hunk 2 above)
Comment (Elchi3) [reply live-examples/js-examples/dataview/dataview-getbigint64.html]: Looks cleaner that way indeed. Updated!

**Comment by wbamberg** [comment]
It's also very cool that this example now shows bigint arithmetic, that you have to use all bigints and can't (AFAIK) mix them with normal numbers, which I think is an unintuitive thing for people, so very much worth showing.

**Pull request merged**